app.py

```diff
@@ -153,6 +153,59 @@ def sent_escape(sent):
 	senttok = sent.split()
 	return " ".join(senttok_escape(senttok))
 
+class ActivedopTree:
+	"""Wrapper for a ParentedTree object with additional methods for activedop.
+	ptree: ParentedTree object with terminals that are numeric indices.
+	senttok: list of tokens corresponding to the terminals of ptree.
+	cgel_tree: CGELTree object.
+	cgel_terminals: list of CGELTree terminals.
+	brackettreestr: string representation of ptree in bracket notation, with labels consisting of a POS tag and a function tag separated by a hyphen.
+	treestr: helpful alias for string representation of the tree (CGEL or bracket notation depending on app settings).
+	gtree: html representation of ptree.
```

Comment:
The method descriptions should be in triple-quoted strings under the function signature.